packages/block-library/src/navigation/edit/use-convert-classic-menu-to-block-menu.js

```diff
@@ -121,13 +138,20 @@ function useConvertClassicToBlockMenu( clientId ) {
 		setStatus( CLASSIC_MENU_CONVERSION_PENDING );
 		setError( null );
 
-		return await convertClassicMenuToBlockMenu( menuId, menuName )
+		return await convertClassicMenuToBlockMenu(
+			menuId,
+			menuName,
+			postStatus
+		)
 			.then( ( navigationMenu ) => {
 				setStatus( CLASSIC_MENU_CONVERSION_SUCCESS );
+				// Reset the ID for the currently importing classic menu.
+				isConvertingClassicMenu = null;
 				return navigationMenu;
 			} )
 			.catch( ( err ) => {
 				setError( err?.message );
+				// Reset the ID for the currently importing classic menu.
```

Comment:
```suggestion
				// Reset the ID for the currently importing classic menu.
				isConvertingClassicMenu = null;
```

I assume we want to reset the menu being converted if it failed to convert?

Reply:
Going to test this

Reply:
Confirmed fixed.